client/src/js/base/__tests__/Tooltip.test.js

```diff
@@ -1,7 +1,11 @@
 import { Tooltip } from "../Tooltip";
 
 describe("<Tooltip />", () => {
-    let props;
+    let props = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['props' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5084784)